book/1-begin/pages/_document.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import Document, { Head, Html, Main, NextScript } from 'next/document';
+import { ServerStyleSheets } from '@material-ui/styles';
+
+class MyDocument extends Document {
+  render() {
+    return (
+      <Html lang="en">
+        <Head>
+          <meta charSet="utf-8" />
+          <meta name="google" content="notranslate" />
+          <meta name="theme-color" content="#1976D2" />
+
+          <link
+            rel="shortcut icon"
+            href="https://storage.googleapis.com/builderbook/favicon32.png"
+          />
+          <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Muli:300,400:latin"
+          />
+          <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+          <link
+            rel="stylesheet"
+            href="https://storage.googleapis.com/builderbook/nprogress.min.css"
+          />
+          <link rel="stylesheet" href="https://storage.googleapis.com/builderbook/vs.min.css" />
+
+          <style>
+            {`
+             a {
+                font-weight: 400;
+                color: #58a6ff;
+                text-decoration: none;
+                outline: none;
+              }
+              blockquote {
+                padding: 0 1em;
+                color: #555;
+                border-left: 0.25em solid #dfe2e5;
+              }
+              pre {
+                display:block;
+                overflow-x:auto;
+                padding:0.5em;
+                background:#FFF;
+                color: #000;
+                border: 1px solid #ddd;
+              }
+              code {
+                font-size: 14px;
+              }
+            `}
+          </style>
+        </Head>
+        <body>
+          <Main />
+          <NextScript />
+        </body>
+      </Html>
+    );
+  }
+}
+
+MyDocument.getInitialProps = async (ctx) => {
+  // Render app and page and get the context of the page with collected side effects.
+  const sheets = new ServerStyleSheets();
```

Comment:
Another here!!!!!!!

Reply:
@jbanahan please note that we've updated code in `builderbook` and are now updating text in the book to match those code changes, in case you notice discrepancies between the text and code.